### PR TITLE
Support multiple query of same entity with different alias

### LINKED ISSUE #994: Elide 5.0: Running Multiple Queries in same transaction is broke in GraphQL
As `EntityProjection` currently requires an entityClass, we can't have multiple queries, where each query has its own entity class, in the same transaction. This wouldn't affect the JSON api endpoint but GraphQL endpoint previously support running multiple queries at the same time.

## Expected Behavior
We should support multiple queries in one request.
```GraphQL
query {
  Book {
    ...
  }
  Author {
    ...
  }
}
```

## Current Behavior
This request above would be rejected by graphql endpoint.

### PR DESCRIPTION
**Pull request by Chandrasekar-Rajasekar**
Resolves #994 (if appropriate)

## Description
We don't want to merge two root level queries on the same entity. We support such queries if they have different alias. Else we reject it. 

Extension of https://github.com/yahoo/elide/pull/1044

## License
I confirm that this contribution is made under an Apache 2.0 license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: elide-graphql/src/test/java/com/yahoo/elide/graphql/GraphQLEndpointTest.java [hunk 1]
```diff
@@ -800,12 +786,7 @@ public void testMultipleRoot() throws JSONException {
                                 "author",
                                 selections(
                                         field("id"),
-                                        field("name")
```
Comment (aklish) [inline elide-graphql/src/test/java/com/yahoo/elide/graphql/GraphQLEndpointTest.java]: This is no longer testing multiple roots.  Let's rename the method or remove the test.

File: elide-graphql/src/test/java/com/yahoo/elide/graphql/GraphQLEndpointTest.java [hunk 2]
```diff
@@ -337,22 +337,8 @@ void testPartialResponse() throws IOException, JSONException {
                 )
         ).toQuery();
 
-        String expectedData = document(
```
Comment (aklish) [inline elide-graphql/src/test/java/com/yahoo/elide/graphql/GraphQLEndpointTest.java]: What about testing the sort parameter for two aliased root level collections?

File: elide-graphql/src/main/java/com/yahoo/elide/graphql/containers/RootContainer.java [hunk 3]
```diff
@@ -14,12 +14,16 @@
 public class RootContainer implements GraphQLContainer {
     @Override
     public Object processFetch(Environment context, PersistentResourceFetcher fetcher) {
+        String entityName = context.field.getName();
+        String aliasName = context.field.getAlias();
+        String keyName = (aliasName == null ? "" : aliasName) + ":" + entityName;
```
Comment (aklish) [inline elide-graphql/src/main/java/com/yahoo/elide/graphql/containers/RootContainer.java]: Let's put this into a static function so the logic won't be duplicated.

File: elide-graphql/src/test/java/com/yahoo/elide/graphql/FetcherFetchTest.java [hunk 4]
```diff
@@ -200,11 +200,6 @@ public void testAliasSameRelationship() throws Exception {
         runComparisonTest("aliasSameRelationship");
     }
 
-    @Test
```
Comment (aklish) [inline elide-graphql/src/test/java/com/yahoo/elide/graphql/FetcherFetchTest.java]: Why did we remove this test?

Comment (Chandrasekar-Rajasekar) [reply elide-graphql/src/test/java/com/yahoo/elide/graphql/FetcherFetchTest.java]: This test checks if there is two root level query of same entity, (without alias) then the response should merge these two queries. We decided to not support these conditions. 
Request- 
`
{
  book(ids: ["1"]) {
    edges {
      node {
        alias1: id
      }
    }
  }
  book(ids: ["1"]) {
    edges {
      node {
        alias2: id
      }
    }
  }
}
`

Response-
`
{
  "book": {
    "edges": [
      {
        "node": {
          "alias1": "1",
          "alias2": "1"
        }
      }
    ]
  }
}
`

File: elide-graphql/src/test/java/com/yahoo/elide/graphql/GraphQLEndpointTest.java (hunk 1 above)
Comment (Chandrasekar-Rajasekar) [reply elide-graphql/src/test/java/com/yahoo/elide/graphql/GraphQLEndpointTest.java]: This one still tests two root level query, one author and one book without alias.

**Pull request merged**